docs/content/guides/6.best-practices/3.ai-contexts.md

```diff
@@ -0,0 +1,69 @@
+# Coding with AI
+
+Alokai aims to boost developer productivity in many ways. E.g. by giving you storefront accelerator and OOTB integrations with 3rd party services.
+Now we want to go even further by boosting your productivity with AI.
+
+## Disclaimer
+
+AI field is evolving rapidly. New models, new tools, new workflows are emerging every day.
+While we do our best to keep this guide updated, we might miss some things.
+Revisit this guide regularly and don't follow it blindly.
+
+## AI Contexts
+
+Alokai provides a set of AI contexts that can be used to code with AI.
+//TODO: explain what are AI contexts
+
+## Expectations Management
+
+//TODO: elaborate on this points
+
+- AI is not a replacement for your knowledge and experience.
+- AI is a tool to help you code faster and more efficiently.
+- AI is not a replacement for your ability to reason and think.
+- AI is just predicts what would be the most likely outcome for the given input.
+There's also some randomness added to the process, so the results might be inconcistent.
+- AI can make silly mistakes. But it can do amazing things at the same time. Don't get disappointed too fast.
+- Don’t extrapolate your vision of intelligence on AI. AI is a different form of intelligence than human intelligence.
```

Comment:
AI doesn't work the same way like human intelligence does. Some things that are easy to humans, are hard for AI, while some things that are hard for humans, are easy for AI.

I think we could also give examples of AI failing to do arithmetics or count "r" in strawberry.